Created Product Page

rares2k opened the pull request:
Created the product page

rares2k committed:
Created Product Page

Created the product page

CimpeanAndreea commented on lib/main.dart:
```diff
@@ -22,7 +23,7 @@ class _MyAppState extends State<MyApp> {
   Widget build(BuildContext context) {
     return MaterialApp(
-        home: LoginPage()
+        home: ProductPage()
```
Revert, will be handled with navigation :D

CimpeanAndreea commented on lib/productpage/AppBar.dart:
```diff
@@ -0,0 +1,69 @@
+    flexibleSpace: Container(
+      decoration: BoxDecoration(
+          image: DecorationImage(
+              image: AssetImage('assets/bluegradient.png'), 
```
Constant for path

CimpeanAndreea commented on lib/productpage/AppBar.dart:
```diff
@@ -0,0 +1,69 @@
+    leading: Container(
+      child: Row(
+        children: [
+          Image.asset('assets/bluelogo.png'),
```
Constant

CimpeanAndreea commented on lib/productpage/productpage.dart:
```diff
@@ -0,0 +1,502 @@
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: buildAppBar(context),
+      backgroundColor: Color(0xE7052C7),
```
Constant

CimpeanAndreea commented on lib/productpage/productpage.dart:
```diff
@@ -0,0 +1,502 @@
+                  Expanded(
+                    flex: 1,
+                    child: Image.asset(
+                      'assets/caramida.png',
```
Constant

CimpeanAndreea commented on lib/productpage/productpage.dart:
```diff
@@ -0,0 +1,502 @@
+                            Padding(
+                              padding: const EdgeInsets.only(bottom: 10),
+                              child: Text(
+                                "Acoperis / Tigla",
```
Is this a category and a subcategory? To me this seems to need support to add multiple categories when data will come from database, these are just hard-coded values. Try to make this generic, extract the hard coded values somewhere else and here pass somehow parameters and let the widget decide how to display these, regardless of what's coming as parameters.

CimpeanAndreea commented on lib/productpage/productpage.dart:
```diff
@@ -0,0 +1,502 @@
+                            Padding(
+                              padding: const EdgeInsets.only(bottom: 10),
+                              child: Text(
+                                "Tigla metalica Bravo, 1540 x 1210 x 0.40 mm, rosu, lucios",
```
Same as the comment above

CimpeanAndreea commented on lib/productpage/productpage.dart:
```diff
@@ -0,0 +1,502 @@
+                                    Padding(
+                                      padding: const EdgeInsets.only(bottom: 20),
+                                      child: Text(
+                                        "Stoc: Disponibil / Indisponibil",
```
Pretty sure it cannot be "Disponibil" and "Indisponibil" at the same time. It also should consider the comment above

CimpeanAndreea commented on lib/productpage/productpage.dart:
```diff
@@ -0,0 +1,502 @@
+                                            ),
+                                          ),
+                                          Icon(
+                                            Icons.star,
```
Is there no way to loop over this? You have the same exact code for the next lines, I wonder how would it look like with 10 stars? :D

CimpeanAndreea commented on lib/productpage/productpage.dart:
```diff
@@ -0,0 +1,502 @@
+                                              ClipRRect(
+                                                borderRadius: BorderRadius.circular(5),
+                                                child: Image.asset(
+                                                  'assets/bluegradient.png',
```
Constant

CimpeanAndreea commented on lib/productpage/productpage.dart:
```diff
@@ -0,0 +1,502 @@
+                                              Padding(
+                                                padding: const EdgeInsets.only(left: 10),
+                                                child: Text(
+                                                  "Preturi mici zilnic"
```
Please find more benefits :D And should be generic as mentioned above. Also the same question, can this be turned into a loop? You have a lot of duplicated code. Think about how many places you would have to modify and be careful of if we want to change the icon to be blue instead of green.

CimpeanAndreea commented on lib/productpage/productpage.dart:
```diff
@@ -0,0 +1,502 @@
+                      ),
+                    ),
+                  ),
+                  Text(
```
Make this generic, to pass the text from outside

CimpeanAndreea commented on lib/productpage/productpage.dart:
```diff
@@ -0,0 +1,502 @@
+          Container(
+            decoration: BoxDecoration(
+                image: DecorationImage(
+              image: AssetImage('assets/bluegradient.png'),
```
Constant

rares2k committed:
Fixed all comments

Changed all the image paths, updated the constants file and resolved all comments.

rares2k committed:
Merge branch 'master' into feature/product_page

rares2k committed:
Changed App Bar

Changed App Bar

rares2k committed:
Deleted AppBar

Changed the appbar and deleted the old one

rares2k committed:
Resolved the conflict in the constants fiel
